project/scenes/game/characters/player/player.gd

```diff
@@ -63,7 +67,7 @@ func raycasts() -> void:
 	else:
 		is_on_wall_r = false
 
-func camera() -> void:
+func camera() -> void:	#not used 
```

Comment:
kaj pomeni not used? če ne rabiš method zbriši

Reply:
bi lahko bil rabljen za debugat animations, je samo deactivated ker ga ne rabimo za zdej, mu dodam boljsi commento